### ISSUE TITLE
"java.io.IOException: $APP_CONFIG not found" when IntelliJ looses focus

### ISSUE THREAD

**Issue by mjiderhamn** [ShyykoSerhiy/gfm-plugin#16]
Love your plugin, but there is some problem with it that unfortunately makes it unusable. As soon as a README.md file has been opened in the editor (closing again doesn't help) and IntelliJ looses focus or you explicitly select `File`, `Save all` from the menu, this error appears:

```
2015-03-11 08:36:45,028 [  93731]   INFO - plication.impl.ApplicationImpl - Saving application settings failed 
com.intellij.openapi.components.StateStorageException: java.io.IOException: $APP_CONFIG not found
	at com.intellij.openapi.components.impl.stores.XmlElementStorage$XmlElementStorageSaveSession.save(XmlElementStorage.java:294)
	at com.intellij.openapi.components.impl.stores.StateStorageManagerImpl$StateStorageSaveSession.save(StateStorageManagerImpl.java:521)
	at com.intellij.openapi.components.impl.stores.ComponentStoreImpl.executeSave(ComponentStoreImpl.java:383)
	at com.intellij.openapi.components.impl.stores.ComponentStoreImpl$SaveSessionImpl.save(ComponentStoreImpl.java:410)
	at com.intellij.openapi.components.impl.stores.StoreUtil.doSave(StoreUtil.java:48)
	at com.intellij.openapi.application.impl.ApplicationImpl._saveSettings(ApplicationImpl.java:1406)
	at com.intellij.openapi.application.impl.ApplicationImpl.saveSettings(ApplicationImpl.java:1445)
	at com.intellij.ide.SaveAndSyncHandlerImpl.saveProjectsAndDocuments(SaveAndSyncHandlerImpl.java:154)
	at com.intellij.ide.SaveAndSyncHandlerImpl$3.onFrameDeactivated(SaveAndSyncHandlerImpl.java:103)
	at com.intellij.ide.FrameStateManagerImpl.a(FrameStateManagerImpl.java:87)
	at com.intellij.ide.FrameStateManagerImpl.access$500(FrameStateManagerImpl.java:32)
	at com.intellij.ide.FrameStateManagerImpl$2$1.run(FrameStateManagerImpl.java:72)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:351)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:318)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:749)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:702)
	at java.awt.EventQueue$3.run(EventQueue.java:696)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:719)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.io.IOException: $APP_CONFIG not found
	at com.intellij.openapi.components.impl.stores.StorageUtil.a(StorageUtil.java:244)
	at com.intellij.openapi.components.impl.stores.StorageUtil.writeFile(StorageUtil.java:180)
	at com.intellij.openapi.components.impl.stores.FileBasedStorage$FileSaveSession.doSave(FileBasedStorage.java:153)
	at com.intellij.openapi.components.impl.stores.XmlElementStorage$XmlElementStorageSaveSession.save(XmlElementStorage.java:290)
	... 31 more
```

**Comment by ShyykoSerhiy** [comment]
Hi, thanks for providing issue:). 
Can you please specify your Intellij version and OS?

**Comment by mjiderhamn** [comment]
IntelliJ Ultimate 14.0.3 running on top of 64-bit Java 1.8.0_31 on Windows 7

**Comment by ShyykoSerhiy** [comment]
It's fixed in v0.0.5. It will take some time for JetBrains to approve it in plugin repo. Until then you can use https://github.com/ShyykoSerhiy/gfm-plugin/raw/master/gfm-plugin.zip. 
Thanks again for issue.
